test/e2e/manifests/mixins/workflow-controller-configmap.yaml

```diff
@@ -13,10 +13,10 @@ data:
     resources:
       requests:
         cpu: 0.1
-        memory: 64Mi
+        memory: 128Mi
```

Comment:
why not just increase the limit but leave the request as is?

Reply:
Sure, can do that.